ingest/Ingest.py

```diff
@@ -0,0 +1,102 @@
+__author__ = 'Hao'
+
+import multiprocessing
+from SPARQLWrapper import SPARQLWrapper, JSON
+import itertools
+import json
+import requests
+
+class Ingest:
+    """say something"""
+
+    def __init__(self, object_index, object_type,
+                 get_objects_query_location, describe_object_query_location, variable_name_sparql):
+        self.object_index = object_index
+        self.object_type = object_type
+        self.records = []
+        self.get_objects_query = Ingest.load_file(get_objects_query_location)
+        self.describe_object_query = Ingest.load_file(describe_object_query_location)
+        self.variable_name_sparql = variable_name_sparql
+
+
+    def load_file(filepath):
```

Comment:
Check if the newest push looks okay. In this way, we can always use pydoc.help to generate the full documentation of it.

(I am not sure if this is the way you want though.. But at least now they are all written down so it should be easy to change ways to present..)